js/apps/admin-ui/public/locales/en/user-federation-help.json

```
@@ -27,6 +27,7 @@
   "ldapSynchronizationSettingsDescription": "This section contains options related to synchronization of users from LDAP to the Keycloak database.",
   "syncRegistrations": "Should newly created users be created within LDAP store? Priority effects which provider is chosen to sync the new user. This setting is effectively appplied only with WRITABLE edit mode.",
   "importUsersHelp": "If true, LDAP users will be imported into the Keycloak DB and synced by the configured sync policies.",
+  "validateUserListingsHelp": "If disabled, import validation will be skipped for all user listing operations e.g. search, users in group, etc.; The validation checks if the user is still present in the federation source with dedicated queries instead of previously cached data. If disabled, then changes made in LDAP are only visible locally after explicit or periodic synchronization.",
```

Since this help text is already in a file suffixed with `-help`, there should be no need to repeat this in the key. I am not sure why for some of these keys this was introduced, but it should not be the case.
```suggestion
  "validateUserListings": "If disabled, import validation will be skipped for all user listing operations e.g. search, users in group, etc.; The validation checks if the user is still present in the federation source with dedicated queries instead of previously cached data. If disabled, then changes made in LDAP are only visible locally after explicit or periodic synchronization.",
```